src/tests/unit/packages/bpmn/bpmn-swimlane/bpmn-swimlane-component-test.tsx

```diff
@@ -0,0 +1,22 @@
+import { wrappedRender } from '../../../test-utils/render';
+import * as React from 'react';
+import { CSSProperties } from 'react';
+import { Multiline } from '../../../../../main/utils/svg/multiline';
+import { BPMNSwimlane } from '../../../../../main/packages/bpmn/bpmn-swimlane/bpmn-swimlane';
+import { BPMNSwimlaneComponent } from '../../../../../main/packages/bpmn/bpmn-swimlane/bpmn-swimlane-component';
+
+// override getStringWidth, because it uses by jsdom unsupported SVGElement methods
+Multiline.prototype.getStringWidth = (str: string, style?: CSSProperties) => {
```

Comment:
same as the other test, I don't need this patch locally to get the tests passing.